feathr_project/feathr/_feature_registry.py

```diff
@@ -736,10 +736,10 @@ def list_registered_features(self, project_name: str = None, limit=50, starting_
                 # split the name based on delimiter
                 result = qualified_name.split(self.registry_delimiter)
                 if result[0].casefold() == project_name:
-                    feature_list.append(entity["name"])
+                    feature_list.append({"name":entity["name"],'id':entity['id'],"qualifiedName":entity['qualifiedName']})
```

Comment:
should we call it GUID or something? to make it more clear.

Reply:
My suggestion is we keep it as it is for now, since changing this might break Blair's UI...  I can change this before his PR merges